src/routers/alpha-router/alpha-router.ts

```diff
@@ -1662,45 +1756,14 @@ export class AlphaRouter
         amount.toExact()
       );
 
-      if (routesToCache) {
```

Comment:
you still want to check that `routesToCache` exists right? if its null we should not even add it. Also, i dont understand how we can end on the state we are looking for above, if we always check that `routesToCache` is not null?

Reply:
oh woops missed that my bad